app/lib/efile/nj/nj1040_calculator.rb

```diff
@@ -271,6 +273,54 @@ def calculate_line_58
         (@direct_file_data.fed_eic * 0.4).round
       end
 
+      def get_personal_excess(ssn, excess_type, threshold)
+        persons_w2s = @intake.state_file_w2s.all&.select { |w2| w2.employee_ssn == ssn }
+        return 0 unless persons_w2s.count > 1
+        return 0 if persons_w2s.any? { |w2| w2[excess_type] && w2[excess_type] > threshold }
+
+        excess_contribution_count = 0
+        total_contribution = 0
+
+        persons_w2s.each do |w2|
+          contribution = w2[excess_type] || 0
+          total_contribution += contribution
+          excess_contribution_count += 1 if contribution.positive?
```

Comment:
[dust/pebble] is tracking this necessary? Aren't we already accounting for this case with the `return 0 if any...` case above?

Reply:
~The .any case above accounts for the need to return if any contribution exceeds the threshold. This is just counting excess contributions that don't exceed the threshold, because if there is only 1 then it's not in excess~

I wrote this out and then realized it's impossible for there to be one contribution that doesn't exceed the threshold itself, but then still have the contributions exceed the threshold collectively. So yeah we can delete this